packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -63,8 +63,10 @@ const LinkCard = styled(Link)`
   display: block;
 `;
 
-const CapitalizeRelativeTime = styled(RelativeTime)`
-  text-transform: capitalize;
+const FirstLetterStyle = styled(Box)`
```

Comment:
Does this actually need its own component? How come? 🤔

Reply:
its works only on block elements

Reply:
TIL. can it be `inline-block` and does that break anything? 🤔

Reply:
yeah why not, I am so in hurry 😖